gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -164,59 +175,84 @@ end
 -- @return table
 -- @realm client
 function LANG.GetUnsafeLanguageTable()
-	return cached_active
+	return cachedActive
+end
+
+---
+-- Caused by a recent rename of the language names, old addons lost compatibility.
+-- To prevent annoyance, a compatibility mode is introduced.
+-- @param string langName The string key identifier of the language
+-- @return string The language name in the new format
+-- @internal
+-- @realm client
+function LANG.GetNameFromAlias(langName)
+	langName = langName and string.lower(langName) or nil
+
+	if LANG.IsLanguage(langName) then
+		return langName
+	end
+
+	for name, tbl in pairs(LANG.Strings) do
+		if tbl.__alias and string.lower(tbl.__alias) == langName then
+			MsgN("[DEPRECATION WARNING]: Language name identifier deprecated, please switch from '" .. langName .. "' to '" .. name .. "'.")
+
+			return name
+		end
+	end
+
+	return langName
 end
 
 ---
--- Returns the reference to a language table if it exists
--- @param string name string key identifier for the language
+-- Returns the reference to a language table if it exists.
+-- @param string langName The string key identifier of the language
 -- @return nil|table
 -- @realm client
-function LANG.GetUnsafeNamed(lang_name)
-	lang_name = lang_name and string.lower(lang_name) or nil
+function LANG.GetUnsafeNamed(langName)
+	langName = LANG.GetNameFromAlias(langName)
```

Comment:
What if `langName` is `nil`? Same below

Reply:
`LANG.GetNameFromAlias(langName)` returns the provided lang name if no alias was found. Therefore returns `nil` if `nil` was provided